util.py

```diff
@@ -24,12 +24,12 @@
     cpp_param_name
 )
 from cs import (
-    cs_escape_keyword, 
-    cs_ignore_service_list, 
-    cs_types_decode, 
-    cs_types_encode, 
+    cs_escape_keyword,
+    cs_ignore_service_list,
+    cs_types_decode,
+    cs_types_encode,
     cs_custom_codec_param_name,
-    cs_init_env
+    cs_sizeof
```

Comment:
Should the .Net guys review this change?

Reply:
There is no behaviour change, in the init method they were just adding this function to the env, which we do as well